.github/workflows/buildAndTest.yml

```diff
@@ -24,6 +24,11 @@ concurrency:
 # Use Cached LLVM to improve build times.
 jobs:
   build-test:
+    env:
+      LLVM_PROJ_DIR: "llvm-project"
```

Comment:
How about expanding this to `LLVM_PROJECT_DIR` given that it only saves a few characters?